packages/accordions/src/elements/components/StepContent.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { forwardRef, useEffect, HTMLAttributes } from 'react';
+import { Transition } from 'react-transition-group';
+import { useCombinedRefs } from '@zendeskgarden/container-utilities';
+import { StyledStepContent } from '../../styled';
+import { useStepContext, useStepperContext } from '../../utils';
+
+export const StepContent = forwardRef<HTMLDivElement, HTMLAttributes<HTMLDivElement>>(
+  (props, ref) => {
+    const contentRef = useCombinedRefs<HTMLDivElement>(ref);
+    const { activeIndex, isHorizontal } = useStepperContext();
+    const { currentStepIndex } = useStepContext();
+    const isActive = currentStepIndex === activeIndex;
+    const isVertical = isHorizontal === false;
```

Comment:
Is this necessary? Could the ternary below use `isHorizontal` directly?

Reply:
I like reading the return statement as "if it is vertical return this JSX", particularly cause `StepContent` serves vertical Steppers. It was a stylistic and reading preference, but I can remove it.